6.bulk_Morphology_Elastic_Network/0.split_data/notebooks/0.split_data_regression.ipynb

```diff
@@ -1,872 +1,912 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    if aggregation == True:
</pre><p>could simplify this to:</p><p>if aggregation:</p><p> </p><p>and the below conditional to:</p><p>if not aggregation:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Agreed, this was changed in another PR but this branch was checked out prior to the merge</p>